integrations/influxdb_metrics_dashboard/README.md

```diff
@@ -0,0 +1,81 @@
+# InfluxDB Metrics Dashboard
+
+## Overview
+
+The InfluxDB Metrics Dashboard creates a Grafana dashboard to visualize existing Timestream for InfluxDB instance performance metrics. The application deploys an EC2 instance running Telegraf to scrape the `/metrics` endpoint of a Timestream for InfluxDB instance, and ingest the scraped metrics to Timestream for LiveAnalytics. After the CloudFormation stack has been deployed, a Lambda function creates a Grafana workspace, and uploads the performance metrics dashboard. The Lambda function is run only once during CDK app initialization.
+
+<img src="./images/architecture.png" alt="drawing" width="800"/>
+
+## Configuration
+
+### Prerequisites
+
+1. If not already installed, install the AWS CDK for Go V2 using the [Getting started with the AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) guide.
+2. If you don't already have a database instance, create a new Timestream for InfluxDB instance with the [Getting started with Timestream for InfluxDB](https://docs.aws.amazon.com/timestream/latest/developerguide/timestream-for-influx-getting-started.html) guide.
+
+### Context options
+
+The following context options are required when deploying the CDK application:
+
+1. **InfluxDBEndpoint**: The URL of the Timestream for InfluxDB endpoint, for example `https://example-1234567890.us-east-1.timestream-influxdb.amazonaws.com:8086/metrics`.
+2. **InfluxDBUsername**: The username used to authenticate with Timestream for InfluxDB instance.
+3. **InfluxDBPassword**: The password used to authenticate with the Timestream for InfluxDB instance.
+4. **VpcId**: The ID for the VPC in which the Timestream for InfluxDB instance has been deployed.
+
+The following context options are optional when deploying the CDK application:
+
+1. **GrafanaWorkspaceName**: The name used for the Grafana worspace. The context defaults to `InfluxDBMetricDashboardWorkspace`.
+2. **TimestreamDatasourceName**: The name of the data source used for populating the dashboard. The context defaults to `Amazon Timestream for LiveAnalytics Sample Data Source`.
+3. **DashboardName**: The name used for the Grafana dashboard. The context defaults to `InfluxDB Performance Dashboard`.
+4. **DatabaseName**: The name of the Timestream database where the InfluxDB metrics are stored. The context defaults to `InfluxDBMetrics`.
+
+## Getting started
+
+To deploy the InfluxDB Metrics Dashboard application, use the following CDK commands, and populate all required context options in the deploy command:
+
+1. Provision AWS environment with the following command:
+```shell
+cdk bootstrap
```

Comment:
This caused an error for me. Are there context options that `cdk bootstrap` needs?

```shell
cdk bootstrap                                                                                                                                                                                                                                      
2024/12/20 15:44:41 VpcId context is required for Telegraf instance
ENOENT: no such file or directory, open 'cdk.out/manifest.json'
```

Reply:
Good thing you found this. You only need to bootstrap once, and I did it before any required context was added. I added all the required context to the command, so let me know if that works on your machine.

Reply:
Works now!